spec/graphql/mutations/customers/update_spec.rb

```diff
@@ -74,6 +75,7 @@
       expect(result_data['invoiceGracePeriod']).to be_nil
       expect(result_data['providerCustomer']['id']).to be_present
       expect(result_data['providerCustomer']['providerCustomerId']).to eq('cu_12345')
+      expect(result_data['providerCustomer']['providerPaymentMethods']).to eq(['card', 'sepa_debit'])
```

Comment:
Style/WordArray: Use `%w` or `%W` for an array of words. (https://rubystyle.guide#percent-w)